pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -612,7 +612,7 @@ class Pulp2ModulemdDefaults(Pulp2to3Content):
     # Unit key fields
     module = models.TextField()
     stream = models.TextField()
-    profiles = JSONField(dict)
```

Comment:
I think maybe this should have been ```JSONField(default=dict)``` all along, and it got mixed up by accident.